webui/react/src/services/apiConfig.ts

```diff
@@ -4,21 +4,31 @@ import queryString from 'query-string';
 import { globalStorage } from 'globalStorage';
 import { serverAddress } from 'routes/utils';
 import * as Api from 'services/api-ts-sdk';
-import {
-  jsonToAgents, jsonToCommands, jsonToDeterminedInfo,
-  jsonToLogin, jsonToLogs, jsonToNotebook, jsonToNotebooks, jsonToShells, jsonToTaskLogs,
-  jsonToTensorboard, jsonToTensorboards, jsonToUsers,
-} from 'services/decoder';
 import * as decoder from 'services/decoder';
 import {
-  CommandIdParams, CreateExperimentParams, CreateNotebookParams, CreateTensorboardParams, DetApi,
-  EmptyParams, ExperimentDetailsParams, ExperimentIdParams, GetExperimentsParams, GetTrialsParams,
-  HttpApi, LoginResponse, LogsParams, PatchExperimentParams, SingleEntityParams, TaskLogsParams,
+  CommandIdParams,
```

Comment:
I like the properties of importing like this but it ends up being too many lines. IMO what I think works better would be cleaner (but less efficient probably) to import everytype as `TP` or something and use that. 

suggestion here we probably should stick to the format we use everywhere else to be consistent